docs/reference/edgectl-download.md

```diff
@@ -0,0 +1,29 @@
+# Downloads
+
+Edge Control is available for macOS, Linux, and Windows.
+
+## macOS
+
+Download by curl command with:
```

Comment:
"Download by curl command" sounds odd.

Shouldn't this be something more like "Download with curl"?

Reply:
(Or "Download with the curl command")

Reply:
fixed